src/Resources/app/administration/src/module/findologic-module/components/findologic-config/findologic-config.html.twig

```diff
@@ -113,6 +107,12 @@
                                     :label="$tc('findologic.settingForm.config.filterPosition.label')"
                                     :helpText="$tc('findologic.settingForm.config.filterPosition.tooltipText')">
                             </sw-single-select>
+                            <sw-switch-field
```

Comment:
Please also commit the built JS in `src/Resources/public/administration/js/fin-search.js`.